etc/telegraf.conf

```diff
@@ -5114,11 +5121,17 @@
 #     "virtualDisk.writeOIO.latest",
 #     "sys.uptime.latest",
 #   ]
-#   # vm_metric_exclude = [] ## Nothing is excluded by default
-#   # vm_instances = true ## true by default
+#   vm_metric_exclude = []
+#
+#   ## If true, gather virtual machine instance metrics.
+#   # vm_instances = true
 #
-#   ## Hosts
-#   ## Typical host metrics (if omitted or empty, all metrics are collected)
+#   ## Inventory path of the hosts to gather metrics on.  If empty, all hosts
+#   ## are included.
+#   ##   ex: host_include = [ "/*/host/**"]
+#   # host_include = []
```

Comment:
Make this path "/**" instead. Gets rid of that permission issue someone ran into a while ago. Requires a change to the default struct initializer in vsphere.go. See #5613